public/resources/remix/index.js

```diff
@@ -21,36 +20,37 @@
 
     var isTouchDevice = 'ontouchstart' in document.documentElement;
     var detailsBar = document.querySelector(".details-bar");
-    detailsBar.setAttribute("style", "");
+    
+    if (detailsBar) {
+      detailsBar.setAttribute("style", "");
+      if (isTouchDevice) {
```

Comment:
What's the experience for you on that device using the current implementation of the remix button?

I think all that happens is that for touch-capable devices, the remix button moves to a different spot and doesn't rely on hovering over it to work, so it's still functional.